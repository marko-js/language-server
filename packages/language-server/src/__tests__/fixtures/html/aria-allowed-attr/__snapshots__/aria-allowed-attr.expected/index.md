## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <div aria-pressed="true">Click me</div>
    |  ^^^ Fix all of the following:
  ARIA attribute is not allowed: aria-pressed="true"
```

