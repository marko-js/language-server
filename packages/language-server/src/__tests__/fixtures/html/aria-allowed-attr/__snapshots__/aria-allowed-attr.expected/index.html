<div data-marko-node-id="0" aria-pressed="true">Click me</div>